bin/spack_cmd.bat

```diff
@@ -54,7 +54,9 @@ if defined py_exe (
     "%py_exe%" "%SPACK_ROOT%\bin\haspywin.py"
 )
 
-set "EDITOR=notepad"
+if not defined EDITOR (
```

Comment:
Can this get pulled out

Reply:
Sure

Reply:
On second thought, I'm going to pull out the `set EDITOR...` line entirely, Spack should not be overwriting user configured settings like this and if we're not adding the check to respect user settings here, we shouldn't be setting anything at all.